docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -137,68 +136,146 @@ To output predicted class probabilities instead of predicted classes, you can us
+### Faster presets or hyperparameters
 
-- Do not specify the `hyperparameter_tune` argument (counterintuitively, hyperparameter tuning is not the best way to spend a limited training time budgets, as model ensembling is often superior). We recommend you only use `hyperparameter_tune` if your goal is to deploy a single model rather than an ensemble.
+Instead of trying to speed up a cumbersome trained model at prediction time, if you know inference latency or memory will be an issue at the outset, then you can adjust the training process accordingly to ensure `fit()` does not produce unwieldy models.
 
-- Do not specify `hyperparameters` argument (allow AutoGluon to adaptively select which models/hyperparameters to use).
+One option is to specify more lightweight `presets`:
 
-- Set `time_limits` to the longest amount of time (in seconds) that you are willing to wait. AutoGluon's predictive performance improves the longer `fit()` is allowed to run.
+```{.python .input}
+presets = ['good_quality_faster_inference_only_refit', 'optimize_for_deployment']
+predictor_light = task.fit(train_data=train_data, label=label_column, eval_metric=metric,
+                           presets=presets, time_limits=30)
+```
 
+Another option is to specify more lightweight hyperparameters:
 
+```{.python .input}
+predictor_light = task.fit(train_data=train_data, label=label_column, eval_metric=metric,
+                           hyperparameters='very_light', time_limits=30)
+```
+
+Here you can set `hyperparameters` to 'light','very_light', 'toy' to obtain progressively smaller (but less accurate) models and predictors. Advanced users may instead try manually specifying particular models' hyperparameters in order to make them faster/smaller.
+
+Finally, you may also exclude specific unwieldy models from being trained at all. Below we exclude models that tend to be slower (K Nearest Neighbors, Neural Network, models with custom larger-than-default  hyperparameters):
+
+```{.python .input}
+excluded_model_types = ['KNN','NN','custom']
+predictor_light = task.fit(train_data=train_data, label=label_column, eval_metric=metric,
+                           excluded_model_types=excluded_model_types, time_limits=30)
+```
 
+If you encounter memory issues: try setting `excluded_model_types = ['KNN','XT','RF']` (or some subset of these models), and add `'ignore_text'` to your `presets` list if there happen to be text fields in your data.
 
+If you encounter disk space issues, make sure to delete all `output_directory` folders from previous previous runs! These can eat up your free space if you call `fit()` many times. If you didn't specify `output_directory`, AutoGluon still automatically saved its models to a folder called: "AutogluonModels/ag-[TIMESTAMP]", where TIMESTAMP records when `fit()` was called, so make sure to also delete these folders if you run low on free space.
```

Comment:
Its probably good to add a section here about deleting models.

`predictor.delete_models(models_to_keep='best', dry_run=False)`

for those who only want to use the predictor for inference going forward.

Additionally: `predictor.save_space()` will also help reduce disk usage.

Can also mention 'optimize_for_deployment' preset which does these two calls automatically during fit().